notifications_utils/sanitise_text.py

```diff
@@ -78,13 +80,15 @@ def encode_char(cls, c):
             return c if c is not None else '?'
 
 
-class SanitiseGSM(SanitiseText):
+class SanitiseGSMandWelsh(SanitiseText):
```

Comment:
As discussed, SanitiseSMS or SanitiseTextMessage or something might be a bit less of a mouthful

Reply:
👍 Changed the name to `SanitiseSMS`